docs/docs/segment-dotnet.md

```diff
@@ -0,0 +1,31 @@
+---
+id: dotnet
+title: Dotnet
+sidebar_label: Dotnet
+---
+
+## What
+
+Display the currently active .NET SDK version.
+
+## Sample Configuration
+
+```json
+{
+  "type": "dotnet",
+  "style": "powerline",
+  "powerline_symbol": "",
+  "foreground": "#000000",
+  "background": "#00ffff",
+  "properties": {
+    "prefix": "  "
```

Comment:
Can you add the textual utf32 representation of the icon? Like `\uF321` for example.

Reply:
I can throw another PR in to fix up the other docs using extended characters if that's interesting.